lib/net/http.rb

```diff
@@ -454,6 +454,8 @@ class HTTPHeaderSyntaxError < StandardError; end
   # - Any other value:
   #   leaves the body and header unchanged.
   #
+  # :include: doc/net-http/whats_here.rdoc
```

Comment:
Why do we do an include here instead of just pasting the contents?

Reply:
Done.